res/openvpn-gui-res-pl.rc

```diff
@@ -296,11 +297,11 @@ CAPTION "Select Certificate"
 FONT 8, "Segoe UI"
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 BEGIN
-    LTEXT "PKCS11 Certificates available:", ID_TXT_PKCS11, 17, 12, 171, 12
+    LTEXT "Dostępny jest certyfikat PKCS11:", ID_TXT_PKCS11, 17, 12, 171, 12
     CONTROL "", ID_LVW_PKCS11, "SysListView32", LVS_REPORT | LVS_SINGLESEL | WS_BORDER | WS_TABSTOP, 17, 25, 171,150
-    PUSHBUTTON "&OK", IDOK, 20, 200, 50, 14, BS_DEFPUSHBUTTON | WS_TABSTOP
-    PUSHBUTTON "&Cancel", IDCANCEL, 90, 200, 50, 14, BS_PUSHBUTTON | WS_TABSTOP
-    PUSHBUTTON "&Retry", IDRETRY, 160, 200, 50, 14, BS_PUSHBUTTON | WS_TABSTOP
+    PUSHBUTTON "OK", IDOK, 20, 200, 50, 14, BS_DEFPUSHBUTTON | WS_TABSTOP
+    PUSHBUTTON "Anuluj", IDCANCEL, 90, 200, 50, 14, BS_PUSHBUTTON | WS_TABSTOP
+    PUSHBUTTON "Odśwież", IDRETRY, 160, 200, 50, 14, BS_PUSHBUTTON | WS_TABSTOP
```

Comment:
It says 'Refresh' not 'Retry' as in original, in Polish should be 'Ponów próbę/Spróbuj ponownie'?

Reply:
Yes, and I agree: that would be preferred form, but I was afraid it won't fit on the button.